src/promptflow/tests/sdk_cli_azure_test/e2etests/test_cli_with_azure.py

```diff
@@ -146,3 +148,17 @@ def test_run_file_with_set(self, pf: PFClient, runtime: str, randstr: Callable[[
         run = pf.runs.get(run=name)
         assert isinstance(run, Run)
         assert run.properties["azureml.promptflow.runtime_name"] == runtime
+
+    def test_azure_cli_ua(self, pf):
+        with environment_variable_overwrite("USER_AGENT", ""):
```

Comment:
We shall accept environment variable: PF_USER_AGENT

like existing one: 
PF_NO_INTERACTIVE_LOGIN = "PF_NO_INTERACTIVE_LOGIN"

Reply:
Do we support both USER_AGENT and PF_USER_AGENT?

Reply:
not sure which part uses the USER_AGENT env, not in my sight

Reply:
maybe it's vscode extension

Reply:
Offline synced, will support both